src/shared/_variables.scss

```diff
@@ -1,2 +1,10 @@
 $large-screen : 1024px;
 $medium-screen : 640px;
+$small-screen : 320px;
+$menuBackgroundColor: #ffffff;
+$menuMainColor: #000000;
```

Comment:
Since those colors belong to the menu, they should be declared in the menu file.